internal/binutils/binutils.go

```diff
@@ -159,13 +232,16 @@ func findExe(cmd string, paths []string) (string, bool) {
 // of a binary.
 func (bu *Binutils) Disasm(file string, start, end uint64, intelSyntax bool) ([]plugin.Inst, error) {
 	b := bu.get()
-	args := []string{"-d", "-C", "--no-show-raw-insn", "-l",
-		fmt.Sprintf("--start-address=%#x", start),
+	if !b.objdumpFound {
+		return nil, fmt.Errorf("couldn't find objdump")
```

Comment:
I think saying "couldn't find objdump" in the error here is somewhat misleading since we didn't try to search for the tool here but the error may make it look like we did. I'd use something like "cannot disasm: no objdump tool available" which might be a bit clearer.

Reply:
Done.